appgate/openapi/openapi.py

```diff
@@ -40,7 +40,13 @@
 SPEC_DIR = f"api_specs/{LATEST_API_VERSION}"
 K8S_API_VERSION = "apiextensions.k8s.io/v1"
 K8S_CRD_KIND = "CustomResourceDefinition"
-LIST_PROPERTIES = {"range", "data", "query", "orderBy", "descending", "filterBy"}
+
+# If the definition doesn't contain these properties, we consider that entity as singleton
+LIST_PROPERTIES: Dict[int, set] = {
+    18: {"range", "data", "query", "orderBy", "descending", "filterBy"},
+    19: {"range", "data", "query", "orderBy", "descending", "filterBy"},
+    20: {"range", "data", "query", "orderBy", "descending", "filterBy", "totalCount"},
+}
```

Comment:
When we switch to e.g. API version 21, would it be a good assumption to use the latest version?
i.e. we only add a new version mapping if something changes.